src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectExtensions.cs

```diff
@@ -16,14 +16,16 @@ public static class OpenIdConnectExtensions
         /// <param name="services"></param>
         /// <returns></returns>
         public static IServiceCollection AddOpenIdConnectAuthentication(this IServiceCollection services)
-            => services.AddOpenIdConnectAuthentication(OpenIdConnectDefaults.AuthenticationScheme, _ => { });
+        {
+            services.AddSingleton<IConfigureOptions<OpenIdConnectOptions>, OpenIdConnectConfigureOptions>();
```

Comment:
Okay. Anyway, this shouldn't be a problem, since the second call to `AddOpenIdConnectAuthentication()` should throw immediately.